packages/trader/src/Modules/Reports/Containers/reports.jsx

```diff
@@ -15,92 +15,95 @@ import { localize } from '@deriv/translations';
-        this.props.toggleReports(false);
-    }
+        return () => {
+            setVisibilityRealityCheck(1);
+            toggleReports(false);
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
 
-    onClickClose = () => this.props.routeBackInApp(this.props.history);
+    const onClickClose = () => routeBackInApp(history);
 
-    handleRouteChange = e => {
-        this.props.history.push(e.target.value);
-    };
+    const handleRouteChange = e => history.push(e.target.value);
 
-    render() {
-        const menu_options = () => {
-            const options = [];
+    const menu_options = () => {
+        const options = [];
 
-            this.props.routes.forEach(route => {
-                options.push({
-                    default: route.default,
-                    icon: route.icon_component,
-                    label: route.getTitle(),
-                    value: route.component,
-                    path: route.path,
-                });
+        routes.forEach(route => {
+            options.push({
+                default: route.default,
+                icon: route.icon_component,
+                label: route.getTitle(),
+                value: route.component,
+                path: route.path,
             });
+        });
 
-            return options;
-        };
+        return options;
+    };
 
-        const { routes, location } = this.props;
-        const selected_route = getSelectedRoute({ routes, pathname: location.pathname });
+    const selected_route = getSelectedRoute({ routes, pathname: location.pathname });
 
-        return (
-            <FadeWrapper
-                is_visible={this.props.is_visible}
-                className='reports-page-wrapper'
-                keyname='reports-page-wrapper'
-            >
-                <div className='reports'>
-                    <PageOverlay header={localize('Reports')} onClickClose={this.onClickClose}>
-                        <DesktopWrapper>
-                            <VerticalTab
-                                alignment='center'
-                                id='report'
-                                is_floating
-                                classNameHeader='reports__tab-header'
-                                current_path={this.props.location.pathname}
-                                is_routed={true}
-                                is_full_width={true}
-                                setVerticalTabIndex={this.props.setTabIndex}
-                                vertical_tab_index={selected_route.default ? 0 : this.props.tab_index}
-                                list={menu_options()}
+    return (
+        <FadeWrapper is_visible={is_visible} className='reports-page-wrapper' keyname='reports-page-wrapper'>
+            <div className='reports'>
+                <PageOverlay header={localize('Reports')} onClickClose={onClickClose}>
+                    <DesktopWrapper>
```

Comment:
is it possible to use isMobile() instead of MobileWrapper and DesktopWrapper?

Reply:
See above.